Source/Documentation/Manual/features-rollingstock.rst

```diff
@@ -621,20 +623,30 @@ Here below a sample of a ``.load-or`` file::
   container with respect to the container shape file coordinates. Unfortunately often such 
   offset is not [0,0,0], which would be advisable for newly produced containers. A simple way to 
   state such offset is to use the ``Show Bounding Info`` of ``Shape Viewer``.
+- "EmptyMassKG" is an optional parameter that defines the tare (weight when empty) of the 
+  container. If the parameter is not present, OR uses a default parameter, specific for that 
+  ContainerType.
+- "MaxMassWhenLoadedKG" is an optional parameter that defines the sum of the tare plus the maximum 
+  allowed payload. As above, if the parameter is not present, OR uses a default parameter, specific for that 
+  ContainerType.
+
 
 Pre-setting a .wag file to accommodate containers
 -------------------------------------------------
 
 As a minimum following block must be present in the .wag file for a double stacker::
 
   ORTSFreightAnims (
+		WagonEmptyWeight ( 12.575t )
```

Comment:
How is this different from the Mass parameter defined in the Wagon section of the .wag file?

Reply:
Mass () in .wag file is an original MSTS parameter. It may or may not include freight weight, depending from what the author of the wagon decided. So it is not a reliable value for the empty wagon weight.
Moreover WagonEmptyWeight has been introduced already in the original FreightAnimations code; so it is already used and it is not possible to remove it now.

Reply:
Thanks for the explanation